agent/config/config.go

```diff
@@ -650,3 +651,11 @@ type Tokens struct {
 	Default     *string `json:"default,omitempty" hcl:"default" mapstructure:"default"`
 	Agent       *string `json:"agent,omitempty" hcl:"agent" mapstructure:"agent"`
 }
+
+type ConfigEntries struct {
+	Bootstrap ConfigEntriesBootstrap `json:"bootstrap,omitempty" hcl:"bootstrap" mapstructure:"bootstrap"`
+}
+
+type ConfigEntriesBootstrap struct {
```

Comment:
nit: if we ever do allow non-bootstrap I guess we'll want to reuse this type a level up too so the name will be wrong. That said it's internal only so I guess not a big deal to rename then. Maybe `ConfigEntriesInner` or something would sidestep that though? Don't really mind though.